packages/g/gdbuspp/monitoring.yaml

```diff
@@ -0,0 +1,8 @@
+---
+# Not tracked. Last checked 2025-06-02.
+releases:
+  id: ~
+
```

Comment:
Codeberg/Forgejo supports RSS feeds for releases and tags: https://codeberg.org/OpenVPN/gdbuspp/tags.rss